packages/input/src/Input.js

```diff
@@ -40,6 +53,19 @@ const Input = ({
     return <InputTextarea {...props} />
   }
 
+  if (submitting) {
```

Comment:
cool, looking good. so this will only work for regular inputs and excludes masked inputs and textareas?

Reply:
In its current incarnation, yeah. I created this mostly as a POC to see if it was something we wanted to move forward with. If we do, I can do it for the other inputs/select.

Reply:
cool. how about we let it ride and see how the implementation goes?